source/Grid/utils/CellSizeAndPositionManager.js

```diff
@@ -92,10 +92,6 @@ export default class CellSizeAndPositionManager {
     return 0;
   }
 
-  _isCellSizeNumeric(): number {
-    return typeof this._cellSize === 'number';
-  }
```

Comment:
I don't feel like you would need to remove this. Is there any specific reason?

Reply:
I think it was due to flow type error